content/en/blog/_posts/2024-05-13-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: blog
+title: "Gateway API v1.1: Service mesh, GRPCRoute, and a whole lot more"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+author: >
+  Richard Belleville (Google),
+  Frank Budinsky (IBM),
+  Arko Dasgupta (Tetrate),
+  Flynn (Buoyant),
+  Candace Holman (Red Hat),
+  John Howard (Solo.io),
+  Christine Kim (Isovalent),
+  Mattia Lavacca (Kong),
+  Keith Mattix (Microsoft),
+  Mike Morris (Microsoft),
+  Rob Scott (Google),
+  Grant Spence (Red Hat),
+  Gina Yeh (Google),
+  and other review and release note contributors.
```

Comment:
How about:
```suggestion
  Gateway API 1.1 release team
```
or the same, but with a link to the team. See https://kubernetes.io/blog/2024/04/17/kubernetes-v1-30-release/ for how we do that.